features/step_definitions/admin_steps.rb

```diff
@@ -399,4 +399,8 @@ def update_settings(settings)
   step %{I post the comment "I loved this" on the work "New Work" as a guest with email "#{email}"}
   step %{I should not see "has been blocked at the owner's request"}
   step %{I should see "Comment created!"}
-end
\ No newline at end of file
+end
+
+Then(/^I visit the last activities item$/) do
```

Comment:
You were right in the scenario itself -- should be `When` because it's an action (visiting a page).